src/Authenticator/TokenAuthenticator.php

```diff
@@ -118,4 +119,14 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
 
         return new Result($user, Result::SUCCESS);
     }
+
+    /**
+     * No-op method.
+     *
+     * @param \Psr\Http\Message\ServerRequestInterface $request A request object.
+     * @return void
+     */
+    public function unauthorizedChallenge(ServerRequestInterface $request)
+    {
```

Comment:
`PersistenceInterface` doesn't require this method, `StatelessInterface` does. Is TokenAuth stateless?